xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -853,10 +901,11 @@ If the look up text is invalid or incomplete UTF-16 text (such as unpaired surro
         <Parameter Name="value" Type="System.Byte[]" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">When the method returns, a byte array that contains the base 64-encoded JSON string if the method succeeds.</param>
```

Comment:
This might be more clear
```suggestion
        <param name="value">If the method succeeds, a byte array that contains the base 64-encoded JSON string.</param>
```